src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You should create a single instance of `ColorSupplier` outside of the method and reuse it.